[2024-01-28 12:44:25,999] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETLProcesoCompleto.conexion_BD manual__2024-01-28T12:44:14.840195+00:00 [queued]>
[2024-01-28 12:44:26,034] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETLProcesoCompleto.conexion_BD manual__2024-01-28T12:44:14.840195+00:00 [queued]>
[2024-01-28 12:44:26,035] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-01-28 12:44:26,036] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2024-01-28 12:44:26,037] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-01-28 12:44:26,277] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2024-01-28 12:44:14.840195+00:00
[2024-01-28 12:44:26,284] {standard_task_runner.py:52} INFO - Started process 3356 to run task
[2024-01-28 12:44:26,289] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETLProcesoCompleto', 'conexion_BD', 'manual__2024-01-28T12:44:14.840195+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/ETLSachetto.py', '--cfg-path', '/tmp/tmpzr7jlksb', '--error-file', '/tmp/tmp9x340mvz']
[2024-01-28 12:44:26,291] {standard_task_runner.py:80} INFO - Job 64: Subtask conexion_BD
[2024-01-28 12:44:26,432] {task_command.py:371} INFO - Running <TaskInstance: ETLProcesoCompleto.conexion_BD manual__2024-01-28T12:44:14.840195+00:00 [running]> on host 1fae8229e5ee
[2024-01-28 12:44:26,543] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-01-28 12:44:26,734] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ignacio Sachetto
AIRFLOW_CTX_DAG_ID=ETLProcesoCompleto
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2024-01-28T12:44:14.840195+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-28T12:44:14.840195+00:00
[2024-01-28 12:44:26,738] {ETLSachetto.py:44} INFO - Conectandose a la BD en la fecha: 2024-01-28 12
[2024-01-28 12:44:28,212] {ETLSachetto.py:52} INFO - Conectado a RedShift Correctamente!
[2024-01-28 12:44:28,214] {python.py:173} INFO - Done. Returned value was: None
[2024-01-28 12:44:28,237] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETLProcesoCompleto, task_id=conexion_BD, execution_date=20240128T124414, start_date=20240128T124426, end_date=20240128T124428
[2024-01-28 12:44:28,635] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-28 12:44:28,961] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
